skt/kernelbuilder.py

```diff
@@ -327,13 +336,53 @@ def make_rpm_repo(self, rpm_files):
 
         return repo_dir
 
+    def find_extras(self, rel_path, callback=None, followlinks=True):
```

Comment:
Given the docstring about `followlinks` says "Set this to True only if you're sure that this won't cause loops", I'm not sure default value of `True` is a good idea. Could you use `False` there so there are no unexpected problems? It would be also easier to reuse the method for other extras that way too.